### ISSUE TITLE
[ios] changing higher playback rate can cause the video to hang while audio still playing

### ISSUE THREAD

**Issue by hamy-acg** [TheWidlarzGroup/react-native-video#1822]
# Bug
 changing higher playback rate when the video is playing can cause the video to hang while the audio still playing.

## Platform

Which player are you experiencing the problem on:
* iOS

## Environment info

React native info output:
```bash
System:
    OS: macOS 10.15.1
    CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Memory: 1.91 GB / 16.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 10.15.3 - ~/.asdf/installs/nodejs/10.15.3/bin/node
    Yarn: 1.17.3 - /usr/local/bin/yarn
    npm: 6.4.1 - ~/.asdf/installs/nodejs/10.15.3/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 13.2, DriverKit 19.0, macOS 10.15, tvOS 13.2, watchOS 6.1
  IDEs:
    Android Studio: 3.5 AI-191.8026.42.35.5791312
    Xcode: 11.2/11B52 - /usr/bin/xcodebuild
  npmPackages:
    react: 16.8.6 => 16.8.6 
    react-native: 0.60.5 => 0.60.5 
  npmGlobalPackages:
    react-native-cli: 2.0.1
```

Library version: tested 5.1.0-alpha1

## Steps To Reproduce

1. play the video
2. change to higher rate
3. the video hang and audio is playing

## Expected behaviour

1. play the video
2. change to higher rate
3. the video and audio are playing

**Comment by hamy-acg** [comment]
it looks like it happens because when the video is playing and the rate is changed, it calls setPaused and  [_player playImmediatelyAtRate:_rate] or [_player play] causes the video to hang.

when I made a change in setPaused by checking _paused, the video works fine
```bash
if (_paused) {
    if (@available(iOS 10.0, *) && !_automaticallyWaitsToMinimizeStalling) {
        [_player playImmediatelyAtRate:_rate];
    } else {
        [_player play];
    }
}
[_player setRate:_rate];
```

**Comment by ktarasenkowow** [comment]
Try to delete `[_player setRate:_rate];` from- (void)setPaused:(BOOL)paused  (node_modules/react-native-video/ios/Video/RCTVideo.m)
